includes/definitions/projector.yaml

```diff
@@ -0,0 +1,8 @@
+os: projector
```

Comment:
I don't think we should call the OS `projector`, since it's unique for Epson

Reply:
good point I will change to epson_projector